common/content/finder.js

```diff
@@ -124,15 +128,27 @@ var Finder = Module("finder", {
 
             // PDF.JS files are different, they use their own messages to communicate the results.
             // So we piggyback the end changes to the findbar when there are any.
-            findbar._original_updateControlState = findbar.updateControlState;
+            findbar._vimpbackup_updateControlState = findbar.updateControlState;
```

Comment:
Is this rename so that we're unlikely to tread on someone else's steps if they were doing a similar thing in another addon?